mappings/net/minecraft/util/function/Lazy.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3528 net/minecraft/util/Lazy
+CLASS net/minecraft/class_3528 net/minecraft/util/function/Lazy
```

Comment:
This is not a function at all

Reply:
It fits with the set of functional objects and patterns, though. Lazy isn't a function, but it works with functions.

Reply:
so is `ThreadLocal` a function? Don't think so, just like `Runnable` isn't always a function

Reply:
`ThreadLocal` or `Runnable` may work with functions, but they aren't functions (more like concurrent stuff).